__tests__/topics.test.js

```diff
@@ -0,0 +1,98 @@
+const request = require('supertest');
+const db = require('../db/connection.js');
+const app = require('../app.js');
+
+describe('/api/topics', () => {
+  describe('we have to get all topics with slug and description', () => {
+    test('200: status code and contain the expected data type and fields', async () => {
+      const res = await request(app).get('/api/topics');
+      expect(res.statusCode).toBe(200);
+      expect(Array.isArray(res.body)).toBe(true);
+      res.body.forEach(topic => {
+        expect(topic).toHaveProperty('slug');
+        expect(typeof topic.slug).toBe('string');
+
+        expect(topic).toHaveProperty('description');
+        expect(typeof topic.description).toBe('string');
+      });
+    });
+  });
+});
+
+describe('/api/articles:/article_id', () => {
+  describe('GET /api/articles/:article_id', () => {
+    test('200: status code and contain the expected data type and fields', async () => {
+      const res = await request(app).get(`/api/articles/33`);
+      expect(res.statusCode).toBe(200);
+      expect(Array.isArray(res.body)).toBe(true);
+      res.body.forEach(article => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
```

Comment:
this could be stricter, you know the article id is going to be 33 in this case and you know what the body/author etc. of the article will be